provision/ansible/playbooks/install-sync-gateway-package.yml

```diff
@@ -36,7 +36,8 @@
 
   tasks:
   - include: tasks/flush-server-buckets.yml
-
+    when: skip_bucketflush == "false"
+    
```

Comment:
Does a python False get correctly converted to "false" here? Just double checking.